arrow-fx-coroutines-test/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm"
+    id "org.jlleitschuh.gradle.ktlint"
+}
+
+apply plugin: 'kotlinx-atomicfu'
+
+apply from: "$SUB_PROJECT"
+apply from: "$DOC_CREATION"
+
+dependencies {
+    api "io.arrow-kt:arrow-core:$VERSION_NAME"
+    api project(":arrow-fx-coroutines")
+    api project(":arrow-fx-coroutines-stream")
```

Comment:
This dependency shouldn't be needed AFAIK. I believe you could move the StreamSpec/Arb classes to arrow-coroutines-stream/test directory and get rid of this dependency + leave arrow-coroutines-test focused only in `arrow-coroutines`

Reply:
Yes, that would be possible. I included this since not all the custom `Arb` instances are easy to write for clients, and I didn't want to include even more modules 🙈